src/app/modules/login/services/azure.ad.b2c.service.ts

```diff
@@ -55,4 +56,13 @@ export class AzureAdB2CService {
   getBearerToken(): string {
     return sessionStorage.getItem('msal.idtoken');
   }
+
+  getUserEmail(): string {
+    console.log(this.msal.getAccount());
```

Comment:
I think you forgot to remove this, did not you?

Reply:
Yep. Good catch!!